pkg/vcs/testdata/linux/config-bisect.pl

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Any idea why this file is .pl and not .sh?

Reply:
This script can be deleted now. I forgot to delete it as part of:
https://github.com/google/syzkaller/commit/8f58e4babeecd6606f4c9729919cc85c470bc422
It mocked config-bisect.pl script in the Linux kernel.